dev-util/debugedit/files/debugedit-5.0-musl-1.2.4.patch

```diff
@@ -0,0 +1,57 @@
+From 187ba161d1d18ad0d675115d8a9eee7ec1790074 Mon Sep 17 00:00:00 2001
+From: Violet Purcell <vimproved@inventati.org>
+Date: Mon, 26 Jun 2023 21:46:20 +0000
+Subject: [PATCH] Fix build failure on musl 1.2.4 due to removal of LFS64
+ compat symbols.
+
+--- a/configure.ac
++++ b/configure.ac
+@@ -81,6 +81,7 @@ AC_CHECK_HEADERS([fcntl.h inttypes.h limits.h malloc.h stddef.h stdint.h stdlib.
+ AC_CHECK_HEADERS([error.h],
+ 		 [AC_DEFINE(HAVE_ERROR_H, 1, [has error.h -- non musl system])])
+ 
++
+ # Checks for typedefs, structures, and compiler characteristics.
+ AC_CHECK_HEADER_STDBOOL
+ AC_C_INLINE
+@@ -150,6 +151,8 @@ else
+ fi
+ AC_SUBST([DEBUG_MACRO_FLAG])
+ 
++AC_DEFINE(_FILE_OFFSET_BITS, 64, [make all interfaces 64 bit by default])
```

Comment:
Can you try `AC_SYS_LARGEFILE` instead?

Reply:
Seems configure.ac actually already has AC_SYS_LARGEFILE.